### PR TITLE
change flag in test to avoid numpy overflow warning in tests

### PR DESCRIPTION
**Pull request by ismael-mendoza**
the maximum allowed values for the current mask type (int32) is 2**31-1

### CODE REVIEW

File: metadetect/lsst/tests/test_lsst_metadetect.py [hunk 1]
```diff
@@ -431,7 +431,7 @@ def test_lsst_metadetect_mfrac_ormask(show=False):
     rng = np.random.RandomState(seed=116)
 
     ntrial = 1
-    flag = 2**31
+    flag = 2**31-1
```
Comment (beckermr) [inline metadetect/lsst/tests/test_lsst_metadetect.py:434]: A flag has to be a power of 2. So this is not the right fix.

**Review by beckermr — CHANGES_REQUESTED** [review]

File: metadetect/lsst/tests/test_lsst_metadetect.py (hunk 1 above)
Comment (ismael-mendoza) [reply metadetect/lsst/tests/test_lsst_metadetect.py]: thanks @beckermr, you are right. I changed the flag to change a single bit

**Review by beckermr — APPROVED** [review]

**Comment by beckermr** [comment]
This fixed the warning. Nice work @ismael-mendoza!

**Comment by arunkannawadi** [comment]
We're still unsure about what this was meant to test. I see a [commit](https://github.com/esheldon/metadetect/blob/736f1689563905bfb5a48428e7d29ea5b7859e2b/metadetect/procflags.py#L20) from Matt here than says flag should not be 2**31 or higher, but 4 days later it was explicitly set to 2**31 here.

**Comment by beckermr** [comment]
Where was it set to 2**31? I cannot find the commit.

**Comment by ismael-mendoza** [comment]
I think in this PR: https://github.com/esheldon/metadetect/pull/155/files

**Comment by beckermr** [comment]
That comment and that commit are unrelated. I think this is just a pure bug.

**Pull request merged**